frontend/src/pages/auth/styles.module.scss

```diff
@@ -2,10 +2,10 @@
   display: flex;
   justify-content: center;
   align-items: center;
-  background-color: #3563e9;
   min-height: 100vh;
-  background-image: url('src/assets/img/auth-page-bg.svg');
-  background-size: cover;
+  background-color: #3563e9;
```

Comment:
```suggestion
  background-color: $blue-dark;
```

Reply:
I believed that it will be changed by props in PageLayout component. But now we've clarified this moment. I've corrected.